internal/manager/validate.go

```diff
@@ -0,0 +1,114 @@
+package manager
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/deckhouse/dmt/internal/module"
+)
+
+func (m *Manager) validateModule(path string) error {
+	var errs error
+	m.errors = m.errors.WithLinterID("module").WithRule("definition-file")
```

Comment:
same thing as in manager.go
```suggestion
	errorList := m.errors.WithLinterID("module").WithRule("definition-file")
```

Reply:
fixed